src/libraries/System.Net.Sockets/tests/FunctionalTests/SendFile.cs

```diff
@@ -100,11 +102,44 @@ public void NotConnected_ThrowsException()
             using (Socket s = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp))
             {
                 Assert.Throws<NotSupportedException>(() => s.SendFile(null));
+                Assert.Throws<NotSupportedException>(() => s.SendFile(null, null, null, TransmitFileOptions.UseDefaultWorkerThread));
+                Assert.Throws<NotSupportedException>(() => s.SendFile(null, ReadOnlySpan<byte>.Empty, ReadOnlySpan<byte>.Empty, TransmitFileOptions.UseDefaultWorkerThread));
                 Assert.Throws<NotSupportedException>(() => s.BeginSendFile(null, null, null));
                 Assert.Throws<NotSupportedException>(() => s.BeginSendFile(null, null, null, TransmitFileOptions.UseDefaultWorkerThread, null, null));
             }
         }
 
+        [ConditionalTheory]
```

Comment:
`[PlatformSpecific]` is enough to make this conditional, `[ConditionalTheory]` is used when you need to evaluate `bool` members at runtime.